lms/static/js/sysadmin/mgmt_commands.js

```diff
@@ -88,14 +88,17 @@
     }
 
     function submitForm(event){
+
+        var form = $('#command-form');
+        var form_data = form.clone();
+        form_data.find('input:not([required])').filter(function() { return $(this).val() == ''; }).remove();
```

Comment:
Btw, this is for removing empty commands, @azimpradhan? Why would there be empty commands in the first place?

also, is `$(this).val()` actually an empty string, or can it be "none" or "false"?